Makefile

```diff
@@ -31,6 +31,13 @@ run-cloudy-mozdef: ## Run the MozDef containers necessary to run in AWS (`cloudy
 	docker-compose -f docker/compose/docker-compose-cloudy-mozdef.yml -p $(NAME) pull
 	docker-compose -f docker/compose/docker-compose-cloudy-mozdef.yml -p $(NAME) up -d
 
+.PHONY: run-env-mozdef
+run-env-mozdef: ## Run the MozDef containers with a user specified env file. Run with make 'run-env-mozdef -e ENV=my.env'
+	$(shell test -f $(ENV) || touch $(ENV))
```

Comment:
I'm not a big fan of writing a file to location within the git repo, so could we change this so the environment file would get referenced directly? This way, we could store the environment file in a location outside of the mozdef project directory. It could look something like (with the addition of using a docker override):

```
.PHONY: run-env-mozdef
run-env-mozdef: ## Run the MozDef containers with a user specified env file (relative or absolute paths). Run with make 'run-env-mozdef -e ENV=my.env'
	ENV_FILE=$(abspath $(ENV)) docker-compose -f docker/compose/docker-compose.yml -f docker/compose/docker-compose-user-env.yml -p $(NAME) up -d
```

Reply:
How does the compose.yml file know about ENV_FILE in this case? Is it just known as a global environmental variable?

Reply:
Gotcha. My branch has been updated. I'm unsure if the make file is correct. I followed your edits but broke up the variable assignment and the docker call. Also I'm unsure if the docker compose pull is required.